docs/user-guide/multiple-ingress.md

```diff
@@ -35,6 +35,8 @@ This mechanism also provides users the ability to run _multiple_ NGINX ingress c
 To do this, the option `--ingress-class` must be changed to a value unique for the cluster within the definition of the replication controller.
 Here is a partial example:
 
+The flags regarding the option `--ingress-class` will be diasbaled soon,instead use `--ingressClassName`
```

Comment:
typo here in disabled word.

Also I would like to hold this, as we are relying on ingress-class now for the ingress-by-name feature